packages/trpc/server/routers/viewer/organizations/create.handler.ts

```diff
@@ -175,7 +170,6 @@ export const createHandler = async ({ input, ctx }: CreateOptions) => {
 
     return { user: { ...createOwnerOrg, password } };
   } else {
-    if (!IS_PRODUCTION) return { checked: true };
```

Comment:
I don't think it's needed. Code verification still accepts any code for dev mode, so there is no need to skip sending email